src/test/java/com/teragrep/pth10/syntaxErrorTest.java

```diff
@@ -59,9 +57,9 @@ public void syntaxTest() {
         String e;
         final String q;
         q = "index = archive_memory ( host = \"localhost\"  Deny";
-        e = "failed to parse at line 1:49 due to missing PARENTHESIS_R at '<EOF>'";
-        Throwable exception = assertThrows(IllegalStateException.class, () -> utils.getQueryAnalysis(q));
-        assertEquals(e, exception.getMessage());
+        e = "Assertions.failed to parse at line 1:49 due to missing PARENTHESIS_R at '<EOF>'";
```

Comment:
Five of the expected exception messages have been accidentally changed here in SyntaxErrorTest.

Reply:
Pushed a fix for all accidental failed -> Assertions.failed conversions